domains/wooferthewoof.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/WooferTheWoof/wooferthewoof.github.io",
+    "owner": {
+        "username": "WooferTheWoof",
+        "email": "ivceball@gmail.com",
+        "discord": "wooferthywoof"
+    },
+    "record": {
+        "CNAME": "WooferTheWoof.github.io"
```

Comment:
Your CNAME redirects back to the domain you are requesting, creating a loop. Unable to see any actual site content. Invalid record.